bankingapp/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+  db:
+    image: postgres
+
+    environment:
+      - POSTGRES_PASSWORD="${DB_PASS}"
+
+    volumes:
+      - data:/var/lib/postgresql/data
+
+
+  bankingapp:
```

Comment:
Since this is the backend, the normal convention is to name the root folder and service as "server"